docs/admin.md

```diff
@@ -0,0 +1,13 @@
+# Admin
+
+## Flags
+
+[`/admin/flags`](https://wpt.fyi/admin/flags) allows site-wide settings to be set, similar to the per-user [`/flags`](https://wpt.fyi/flags).
+
+## Flushing caches
+
+[`/admin/cache/flush`](https://wpt.fyi//admin/cache/flush) flushes some caches.
```

Comment:
@Hexcles does just visiting this do something? I didn't try out of fear of side effects.

Reply:
Yes, which is probably a terrible design...